packages/esm-ugandaemr-app/src/views/home/header/ugemr-home-header.component.tsx

```diff
@@ -0,0 +1,68 @@
+import { Dropdown } from '@carbon/react';
+import { Calendar, Location } from '@carbon/react/icons';
+import { formatDate, useSession } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  updateSelectedQueueLocationName,
+  updateSelectedQueueLocationUuid,
+  updateSelectedServiceName,
+  useSelectedFacilityName,
+  useSelectedQueueLocationName,
+} from '../../../../../esm-patient-queues-app/src/helpers/helpers';
+import PatientQueueIllustration from '../../../../../esm-patient-queues-app/src/patient-queue-header/patient-queue-illustration.component';
+import { useQueueLocations } from '../../../../../esm-patient-queues-app/src/patient-search/hooks/useQueueLocations';
+import styles from './patient-home-header.scss';
+
+const PatientHomeHeader: React.FC<{ title?: string }> = ({ title }) => {
```

Comment:
Typically it's a good practice to have the component name similar to the file unless there many exported components within the same component.